src/components/ErrorScreen.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
+import { Box, Button, Center, Flex, Text, Icon } from "@chakra-ui/react";
+import { MdSettings } from "react-icons/md";
+import { useNavigate } from 'react-router-dom';
+
+interface Props {
+  error: Error | undefined;
+}
+
+const ErrorScreen: React.FC = () => {
+  const navigate = useNavigate();
+  const navigateToHome = () => {
+    navigate('');
+  };
```

Comment:
You don't need this helper function, you can just do, `onClick={() => navigate("/")`. Also rn, this button isn't working, I think you need to use `("/")`